cmd/use.go

```diff
@@ -52,8 +52,21 @@ func UseKubectlBinary(version string) error {
 
 	_, err = os.Stat(kubectlVersion)
 	if os.IsNotExist(err) {
-		log.Printf("kubectl %s does not exist", version)
-		os.Exit(1)
+		prompt := promptui.Prompt{
+			// no idea why sprintf doesn't work
+			Label:     fmt.Sprintf("kubectl %s is not installed. Install now", version),
```

Comment:
Is sprintf now working considering the comment?

Reply:
Yep will remove comment - it wasn't rendering properly because I had a newline char in there.